### DISCUSSION TITLE
Migration from TrueNAS to docker-compose

### DISCUSSION THREAD

**Discussion by isZYKerman** [immich-app/immich#13594] [Q&A]
Hi everyone,
I have migrated my storage from TrueNAS to Ubuntu, therefore all the services need to be re-configurated. On TrueNAS six folders (datasets) were required: main library, upload location, thunbnails, video folder, profile, postgres. But the official document of installing via docker-compose only tells me how to bind two of them on the host machine: upload and prostgres. Can someone tell me how to bind the other four directories with the container? I really need this, for the albums in Immich are very important to me.

**Comment by vasi786** [comment]
Hi, did you migrate successfully? I am in the same situation.

**Reply by isZYKerman** [reply]
Yes, I achieved that several months ago...... You can refer to [this](https://immich.app/docs/install/truenas) to help you fill in the "volumes" section in the docker compose file.
